src/docs/perf/app-size.md

```diff
@@ -19,83 +18,68 @@ or by clicking the **Play** button in your IDE
 generates a _debug_ build of the Flutter app.
 The app size of a debug build is large due to
 the debugging overhead that allows for hot reload
-and source level debugging.
+and source level debugging. As such, it is not representative of a production
+app end users download.
 
-To get a better sense of what an end-user actually
-has to download for a Flutter app, use the
-following instructions.
+## Checking the total size
 
-## Android
+A default release build, such as one created by `flutter build apk` or
+`flutter build ios`, is built to conveniently assemble your upload package
+to the Play Store and App Store. As such, they're also not representative of
+your end-users' download size. The stores generally reprocess and split
+your upload package to target the specific downloader and the downloader's
+hardware, such as filtering for assets targeting the phone's DPI, filtering
+native libraries targeting the phone's CPU architecture.
 
-Run the following from the top of your Flutter project
-to get the size of APK for 32-bit Android devices:
+### Estimating total size
 
-```shell
-flutter build apk --target-platform=android-arm
-```
+To get the closest approximate size on each platform, use the following
+instructions.
 
-The output should look something like the following:
+#### Android
 
-```shell
-Built build/app/outputs/apk/release/app-release.apk (4.2MB).
-```
+Follow the Google [Play Console's instructions][] for checking app download and
+install sizes.
 
-For 64-bit Android devices, run the following:
+Produce an upload package for your application:
 
 ```shell
-flutter build apk --target-platform=android-arm64
+flutter build appbundle
 ```
 
-Here is an example output:
+Log into your [Google Play Console][]. Upload your application binary by drag
+dropping the .aab file.
 
-```shell
-Built build/app/outputs/apk/release/app-release.apk (4.6MB).
-```
-
-You can run the following to get 2 APKs,
-one for 32-bit and one for 64-bit:
+View the application's donwload and install size in the 'Android vitals' ->
```

Comment:
donwload => download

Also, for menu items, we use bold, not code font. So:

=> **Android vitals** -> **App size** tab.